File: PHP/SplitStrings/MySolution.php
<?php
function solution($str) {
  $result = $str === "" ? [] : str_split($str, 2);    
  if (strlen(end($result)) == 1){
    $result[key($result)] .= "_"; 
  }
  return $result;
}

// Original kata: https://www.codewars.com/kata/515de9ae9dcfc28eb6000001
// My solution: https://www.codewars.com/kata/reviews/57cb045535ef62f92d0000f1/groups/67a1e9342f61c78fe59c2f8a

File: CS/WelcomeToTheCity/MySolution.cs
public class Kata
{
  public static string SayHello(string[] name, string city, string state)
  {
    return "Hello, " + string.Join(" ", name) + "! Welcome to " + city + ", " + state + "!";
  }
}

// Original kata: https://www.codewars.com/kata/5302d846be2a9189af0001e4
// My solution: https://www.codewars.com/kata/reviews/599c8dc5aa7911eb11002c58/groups/599d6611cd378f48b9000b5e
